openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/TableProfiler.component.tsx

```diff
@@ -120,84 +147,128 @@ const TableProfiler = ({ tableProfiles, columns }: Props) => {
+                              <span className="tw-mx-1">-</span>
+                              <span className="tw-mx-1">{m.value}</span>
+                            </p>
+                          ))}
+                      </div>
+                    </td>
+                    <td
+                      className="tw-relative tableBody-cell"
+                      colSpan={2}
+                      data-testid="tableBody-cell">
+                      <div className="tw-flex tw-justify-between">
+                        {col.columnTests ? (
+                          <div className="tw-border tw-border-main tw-rounded tw-p-2 tw-min-h-32 tw-max-h-44 tw-overflow-y-auto tw-flex-1">
+                            {col.columnTests.map((m, i) => (
+                              <div className="tw-flex tw-mb-2" key={i}>
+                                <p className="tw-mr-2">
+                                  {m.results?.every(
+                                    (result) =>
+                                      result.testCaseStatus === 'Success'
+                                  ) ? (
+                                    <i className="fas fa-check-square tw-text-status-success" />
+                                  ) : (
+                                    <i className="fas fa-times tw-text-status-failed" />
+                                  )}
+                                </p>
+                                <div>
+                                  <span className="tw-mx-1 tw-font-medium">
+                                    {m.testCase.columnTestType}
+                                  </span>
+                                  <div className="tw-mx-1">
+                                    {Object.entries(
+                                      m.testCase.config ?? {}
+                                    ).map((config, i) => (
+                                      <p className="tw-mx-1" key={i}>
+                                        <span>{config[0]}:</span>
+                                        <span>{config[1] ?? 'null'}</span>
+                                      </p>
+                                    ))}
+                                  </div>
+                                </div>
+                              </div>
+                            ))}
+                          </div>
+                        ) : (
+                          `No tests available`
+                        )}
+                        <div className="tw-self-center tw-ml-5">
+                          <Button
+                            className="tw-px-2 tw-py-0.5 tw-rounded tw-border-grey-muted"
+                            size="custom"
+                            type="button"
+                            variant="outlined"
+                            onClick={() => quilityTestFormHandler(6, 'column')}>
+                            Add Test
+                          </Button>
```

Comment:
We must restrict the non-owner and non-admin person for adding test case. and also i think it should have name `qualityTestFormHandler`